Static/js/board.js

```diff
@@ -1789,40 +1931,37 @@ var board = {
 	//   * the initial TPS tagname is optional
 	//   * the player & move elements are optional
 	loadtps: function (tps) {
-		var playerTurn;
-		var moveNumber;
+		let playerTurn;
+		let moveNumber;
 
 		// simple RegEx for a basic TPS notation,
 		// but doesn't specify the details of the actual layout
-		var tpsRE = /\[(TPS\s*)?\"?\s*([,x12345678SC\/]+)(\s+([\d+]))?(\s+(\d+|-))?\s*\"?\s*\]/;
-		var result = tpsRE.exec(tps);
+		const tpsRE = /\[(TPS\s*)?"?\s*([,x12345678SC/]+)(\s+([\d+]))?(\s+(\d+|-))?\s*"?\s*\]/;
```

Comment:
Could we maybe not get rid of "unnecessary" escapes? Seeing / unescaped in the middle of an expression gives me the creeps.